include/libremidi/backends/jack/helpers.hpp

```diff
@@ -13,6 +13,7 @@
 
 #include <atomic>
 #include <semaphore>
+#include <cassert>
```

Comment:
To sort?

Reply:
Compilation failure without that include. Whereever you call assert you get a missing function error. I'm not at my pc but can paste the exact lines if you want later. It's probably being pulled in via another nested include on  the compiler you are building with.